src/OversetSimulation.cpp

```diff
@@ -93,6 +93,10 @@ void OversetSimulation::perform_overset_connectivity()
     m_timers_tg.tick("Connectivity");
     if (m_has_amr) m_tg.preprocess_amr_data();
     m_tg.profile();
+    if (m_holemap_alg == 1 && m_complementary_comm_initialized == false) {
```

Comment:
would you mind doing `((m_holemap_alg == 1) && (m_complementary_comm_initialized == false))` for clarity?

Reply:
I just pushed this change.